CHANGES.rst

```diff
@@ -1,6 +1,13 @@
 Changes
 =======
 
+Version 0.8.1 (2022-02-01)
+---------------------------
+
+- Adds logic to calculate the CPU quota periodically.
```

Comment:
Adds an option to periodically calculate CPU quota usage.  _(to soften the message that this is just an option)_